data/core/units/monsters/Shadow_Jumping_Spider.cfg

```diff
@@ -26,9 +26,9 @@
     cost=58
     usage=fighter
     undead_variation=spider
-    description= _ "Unlike the feared Giant Spider, that uses poison and webs to catch prey, the Shadow Jumper relies on speed and the element of surprise.  These stealthy, keen-eyed hunters creep to within striking distance, then pounce with a piercing bite that is said to punch through steel.  Once they have pierced their victim's defenses, the spiders take what nourishment is available, feeding in a manner as fast and remorseless as befits their nature.
+    description= _ "Unlike the feared Giant Spider that uses poison and webs to catch prey, the Shadow Jumper relies on speed and the element of surprise.  These stealthy, keen-eyed hunters creep to within striking distance, then pounce with a piercing bite that is said to punch through steel.  Once they have pierced their victim's defenses, the spiders take what nourishment is available, feeding in a manner as fast and remorseless as befits their nature.
```

Comment:
Double-spaces and typography in `victim's`. If you're making a change, may as well fix it all.

Reply:
Of course, just demonstrates how difficult this is, I don't get everything in one go either. I'll do this in #7965.